src/Icicle/Avalanche/Prim/Eval.hs

```diff
@@ -111,7 +111,7 @@ evalPrim p vs
       -> primError
 
 
-     PrimUpdate (PrimUpdateArrayPut t)
+     PrimArray (PrimArrayPutMutable t)
```

Comment:
yeah, because this isn't really mutating I can't rely on the Avalanche evaluator for tests